### PR TITLE
AB Micro: delete image

### PR DESCRIPTION
**Pull request by annagav**
#### What are the relevant tickets?

Depends on #711 
#### What's this PR do?

Add a little 'x' icon to every image, that removes that image for chosen sample.

### CODE REVIEW

File: instructor/common.py [hunk 1]
```diff
@@ -1801,6 +1801,38 @@ def add_image_group(request):
...
+    group_id = request.POST.get('group_id')
+    filter_name = request.POST.get('filter')
+    image_id = request.POST.get('image_id')
+    if group_id:
+        image_group = get_object_or_404(
+            models.MicroscopyGroupedImages,
+            id=group_id,
+            image_mapping__sample_prep__assignment__pk=pk
+        )
+        setattr(image_group, filter_name + '_filter_image', None)
```
Comment (noisecapella) [inline instructor/common.py:1823]: Same comment as #710 about needing a whitelist for `filter_name`

**Comment by noisecapella** [comment]
Functionality works fine

**Comment by annagav** [comment]
Addressed comments, @noisecapella.

**Comment by noisecapella** [comment]
:+1:

**Pull request merged**